src/pcapi/core/bookings/api.py

```diff
@@ -119,6 +119,18 @@ def cancel_booking_by_offerer(booking: Booking) -> None:
         redis.add_offer_id(client=app.redis_client, offer_id=booking.stock.offerId)
 
 
+def cancel_booking_for_fraud(booking: Booking) -> None:
+    booking.isCancelled = True
+    booking.cancellationReason = BookingCancellationReasons.FRAUD
+    repository.save(booking)
+    logger.info("Cancelling booking for fraud reason: %s", booking.id)
```

Comment:
```suggestion
    logger.info("Cancelled booking for fraud reason", extra={"booking": booking.id})
```

(J'ai une PR en cours qui reprend les logs de ce module, et c'est le format utilisé.)